instrumentation/apache-httpasyncclient-4.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpasyncclient/ApacheHttpAsyncClientInstrumentation.java

```diff
@@ -111,7 +110,7 @@ public HttpHost getTarget() {
     @Override
     public HttpRequest generateRequest() throws IOException, HttpException {
       HttpRequest request = delegate.generateRequest();
-      GlobalOpenTelemetry.getPropagators()
+      tracer().getPropagators()
```

Comment:
I updated this particular instrumentation because I wanted to make sure that the basic idea was workable.